Core/StateManager.cs

```diff
@@ -32,59 +31,24 @@ public static string SaveToJson()
 
 		public static void LoadTransactions(IEnumerable<(string key, Stream stream)> files)
 		{
-			var transactions = new List<Transaction>();
+			var newTransactions = new List<Transaction>();
 			foreach (var file in files)
 			{
-				transactions.AddRange(importers[file.key].Load(file.stream));
+				newTransactions.AddRange(importers[file.key].Load(file.stream));
 			}
 
-			foreach (var t in transactions)
-			{
-				State.Instance.Transactions.Add(t);
-			}
-		}
-		
-		public static SmoothedTimeSeries GetSmoothedTimeSeries(string category, double smoothingRatio)
-		{
-			var filteredTransactions = State.Instance.Transactions.Where(State.Instance.CategoryFilters[category]).ToList();
-			return new SmoothedTimeSeries(filteredTransactions, smoothingRatio);
-		}
+			var newCategories = newTransactions.Select(t => t.Category).Where(c => c is { } && State.Instance.Categories.All(sc => sc.Name != c))
+				.Distinct().Select(c => new Category(c, new Rule[] { new Rule(".*", "*", ".*", c) }, 1, 10000)).ToList();
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)